client/components/status/status.js

```diff
@@ -211,23 +221,32 @@ export default class Status extends React.Component {
       captivePortalLogoutForm,
       isAuthenticated,
     } = this.props;
-    const {content, links, buttons} = statusPage;
-    const {username, password, sessions} = this.state;
+    const {content, links, buttons, user_info} = statusPage;
+    const {username, password, sessions, userInfo} = this.state;
     const contentArr = getText(content, language).split("\n");
+    userInfo.status = getText(user_info.status.value, language);
     return (
       <>
         <div className="container content" id="status">
           <div className="inner">
             <div className="main-column">
               {contentArr.map(text => {
-                if (text !== "")
-                  return (
-                    <p key={text}>
-                      {text}
-                    </p>
-                  );
-                return null;
-              })}
+                  if (text !== "")
+                    return (
+                      <p key={text}>
+                        {text}
+                      </p>
+                    );
+                  return null;
+                })}
```

Comment:
why did you increase the indentation here? It looks like an unnecessary change and I think it should be reverted.